### DISCUSSION TITLE
Clipboard: wl-copy / wl-paste

### DISCUSSION THREAD

**Discussion by mrusme** [andmarti1424/sc-im#681] [Q&A]
I've browsed through the issues here, trying to find an example that works but it seems I'm out of luck, hence this issue.

My `~/.config/sc-im/scimrc` looks like this:

```
set default_copy_to_clipboard_cmd="wl-copy -t text <"
set default_paste_from_clipboard_cmd="wl-paste -t text"
set copy_to_clipboard_delimited_tab=1
```

Copying doesn't seem to work unfortunately. I've tried removing the quotes, without success - sc-im will show a syntax error without them.

Any idea how to get this working with Wayland's wl-copy/wl-paste commands? Thank you!

**Comment by andmarti1424** [comment]
hello. cannot try that exactly since i dont have wayland right now.. if you issue `echo algo | wl-copy -t text`.. does that word get copied to your clipboard?

**Reply by mrusme** [reply]
`echo "hello" | wl-copy -n` copies `hello` (without the \n that echo appends) to the clipboard, yes. I tried different variation, with `-t text` and without, with `-n` and without, with the `<` at the end and without, nothing seems to be working. I even double checked whether I might be pressing the wrong key (`yy` or `vy`) but it seems shortcuts are identical to vim.

**Reply by andmarti1424** [reply]
what do you mean with `yy` or `vy`? you need to use `:ccopy` to copy to clipboard

**Reply by mrusme** [reply]
https://github.com/andmarti1424/sc-im#quick-start

> yy | Copy current cell

**Reply by andmarti1424** [reply]
That copies to buffer, not to clipboard. Please take a look at doc `:help`

**Reply by mrusme** [reply]
Okay, then functionality is slightly different to vim. Using :ccopy works.

**Comment by andmarti1424** [comment]
i move this to a conversation.
